fe/fe-core/src/main/java/com/starrocks/common/proc/StreamLoadsProcDir.java

```diff
@@ -41,6 +41,7 @@ public class StreamLoadsProcDir implements ProcDirInterface {
                     .add("DbName")
                     .add("TableName")
                     .add("State")
+                    .add("Warehouse")
```

Comment:
‌‌I suggest that we should maintain consistency with the previous logic, and add the display of this column when the cluster is running in shared-data mode.